agnpy/absorption/absorption.py

```diff
@@ -386,6 +473,27 @@ def tau(self, nu):
         if isinstance(self.target, RingDustTorus):
             return self.tau_dt(nu)
 
+    def tau_mu_s(self, nu):
```

Comment:
maybe we can include this in `tau` and call this function only when `mu_s != 0` or not `None`

Reply:
makes sense, I can do the change

Reply:
I modified the function tau to have:
```
if self.mu_s==1: # default value
    ...
    if isinstance(self.target, RingDustTorus):
        return self.tau_dt(nu)
else:
    ...
    if isinstance(self.target, RingDustTorus):
        return self.tau_dt_mu_s(nu)
```
but then the test on test_abs_dt_vs_point_source fails and very likely because the point source behind the blob has the same problem with varying angle between photons along the integral 
I can make a similar function for computing tau for point source and arbitrary tau (the current one would have be be removed then, because it is only valid for mu_s =1 in which case for a point source we get tau=0, this is not a big problem.
However when I do so I'm pretty sure I will get the BLR vs point source failing as well, and computing the BLR absorption  is more tricky because I need to check some geometry relations first, so in the end I will have to disable at least that test

Reply:
done in the latest commit (15d39d9).
as explained above I had to disable one of the tests, we can reenable it once the BLR absorption function is rewritten in a similar way (but most probably I can take care or it only next week, so if you are happy with the current version just merge it and I will make another PR for the BLR ).